src/main/resources/sql/insertDiscreteGroundWater.sql

```diff
@@ -9,21 +9,22 @@ into discrete_ground_water_aqts(monitoring_location_id, agency_code, agency, sit
                                 level_accuracy_code, level_accuracy, measurement_source_code, measurement_source,
                                 measurement_method_code, measurement_method, approval_status_code, approval_status,
                                 result_measure_qualifiers)
-select monitoring_location.monitoring_location_id,
-       monitoring_location.agency_cd agency_code,
-       monitoring_location.agency,
-       monitoring_location.site_identification_number,
-       monitoring_location.monitoring_location_identifier,
-       monitoring_location.site_tp_cd site_type_code,
-       monitoring_location.site_type,
-       monitoring_location.geom,
-       monitoring_location.decimal_latitude,
-       monitoring_location.decimal_longitude,
-       monitoring_location.decimal_latitude_longitude_datum,
-       monitoring_location.well_depth,
-       monitoring_location.hole_depth,
-       monitoring_location.local_aquifer,
-       monitoring_location.local_aquifer_type,
+select null monitoring_location_id,
+       sitefile.agency_cd agency_code,
+       agency.name agency,
+       sitefile.site_no site_identification_number,
+       coalesce(sitefile.agency_cd, '') || '-' || coalesce(sitefile.site_no, '') monitoring_location_identifier,
```

Comment:
I think we get the monitoring location identifier already earlier in the ETL, but now that I think about it, those ids are only relevant to the location identifiers (publishable only) from AQTS, and this task to be able to join on non-publishable records as well, hence the site file join.